CHANGELOG.md

```diff
@@ -4,6 +4,26 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v13.1.0](https://github.com/voxpupuli/puppet-rabbitmq/tree/v13.1.0) (2023-10-30)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-rabbitmq/compare/v13.0.0...v13.1.0)
+
+**Implemented enhancements:**
+
+- Add additional applyto options for policies [\#948](https://github.com/voxpupuli/puppet-rabbitmq/pull/948) ([wyardley](https://github.com/wyardley))
+- Add Puppet 8 support [\#938](https://github.com/voxpupuli/puppet-rabbitmq/pull/938) ([bastelfreak](https://github.com/bastelfreak))
+
+**Fixed bugs:**
+
+- fix purge rabbitmq\_parameter [\#945](https://github.com/voxpupuli/puppet-rabbitmq/pull/945) ([fatpat](https://github.com/fatpat))
+
+**Closed issues:**
+
+- rabbitmq::install::rabbitmqadmin keeps installing python package [\#941](https://github.com/voxpupuli/puppet-rabbitmq/issues/941)
+- rabbitmq\_queue takes a long time when a large number of vhosts is present [\#940](https://github.com/voxpupuli/puppet-rabbitmq/issues/940)
+- Puppet fails to run with an error "no parameter named 'download\_options'" [\#790](https://github.com/voxpupuli/puppet-rabbitmq/issues/790)
+- Module always installs RabbitMQ 3.5.7 even if pinned at 3.6 [\#756](https://github.com/voxpupuli/puppet-rabbitmq/issues/756)
```

Comment:
I added the skip-changelog label because those issue don't bring any value to the release. please regenerate the changelog.

Reply:
Cool -- updated.

I also already added that label to #682 since it shows up under 'enhancements' from having the 'enhancement' label, even though I updated it to closed as "not planned".